### ISSUE TITLE
Build failure: kdePackages.kasts

### ISSUE THREAD

**Issue by bezmuth** [NixOS/nixpkgs#306569]
### Steps To Reproduce

1. Add kdePackages.kasts to system packages
2. Rebuild system

### Build log

```
warning: The interpretation of store paths arguments ending in `.drv` recently changed. If this command is now failing try again with '/nix/store/inmr89ddw9fldc63w4bkzswxplxk0mxf-kasts-24.02.2.drv^*'
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/cai24xk67widrv35wbz520achcw2047l-kasts-24.02.2.tar.xz
source root is kasts-24.02.2
setting SOURCE_DATE_EPOCH to timestamp 1712591645 of file kasts-24.02.2/src/utils/updatefeedjob.h
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/0312bbblig58dld19v6ksj40gqcm6f43-kasts-24.02.2/shar>
-- The C compiler identification is GNU 13.2.0
-- The CXX compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/4kwk49k8rc8cg8c387kbbjcrl8i0y693-gcc-wrapper-13.2.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/4kwk49k8rc8cg8c387kbbjcrl8i0y693-gcc-wrapper-13.2.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Installing in /nix/store/0312bbblig58dld19v6ksj40gqcm6f43-kasts-24.02.2. Run /build/kasts-24.02.2/build/prefix.sh to set the environment for kasts.
-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
CMake Error at /nix/store/0xpv46mb188z226j7cs211zmmi5xw9nj-cmake-3.29.1/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_EXECUTABLE Interpreter)
Call Stack (most recent call first):
  /nix/store/0xpv46mb188z226j7cs211zmmi5xw9nj-cmake-3.29.1/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/0xpv46mb188z226j7cs211zmmi5xw9nj-cmake-3.29.1/share/cmake-3.29/Modules/FindPython/Support.cmake:3867 (find_package_handle_standard_args)
  /nix/store/0xpv46mb188z226j7cs211zmmi5xw9nj-cmake-3.29.1/share/cmake-3.29/Modules/FindPython3.cmake:545 (include)
  /nix/store/1hbyb1zp9vls959jbqdqkjjszy3bbj86-extra-cmake-modules-6.1.0/share/ECM/modules/ECMAddAndroidApk.cmake:57 (find_package)
  CMakeLists.txt:39 (include)


-- Configuring incomplete, errors occurred!
```

### Additional context

Package built fine a few days ago, I would look into it myself but the kde build system within nix seems a bit out of my wheelhouse.

### Notify maintainers

@samueldr

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.4-zen1, NixOS, 24.05 (Uakari), 24.05pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by bezmuth** [comment]
I've tried this both in system packages and home-manager packages.

**Comment by eclairevoyant** [comment]
This was fixed in #305692, just need to wait for it to land.
